dags/analyzer_helper/discord/discord_transform_raw_data.py

```diff
@@ -0,0 +1,220 @@
+import logging
+from typing import Any, Dict, List
+
+from analyzer_helper.discord.transform_raw_data_base import TransformRawDataBase
+from analyzer_helper.discord.utils.is_user_bot import UserBotChecker
+
+
+class DiscordTransformRawData(TransformRawDataBase):
+    def __init__(self, platform_id: str):
+        """
+        Initializes the class with a specific platform ID and sets up a bot checker for user validation.
+        """
+        self.platform_id = platform_id
+        self.user_bot_checker = UserBotChecker(platform_id)
```

Comment:
Does the `UserBotChecker` accept guild_id instead of platform_id? If so, please consider updating it to feed guild_id to it.